dask/array/overlap.py

```diff
@@ -422,6 +422,38 @@ def boundaries(x, depth=None, kind=None):
     return x
 
 
+def ensure_minimum_chunksize(size, chunks):
+    """Determine new chunks to ensure that every chunk >= size"""
```

Comment:
With finicky algorithmic functions like this I find that having one simple example in the docstring does wonders for future maintainers.  Even though it's internal and users won't see it, seeing a quick docstring example can be a great accelerant to understanding for the next person who has to deal with the overlap code.